ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2436,7 +2436,7 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
-- as: gcp-ipi-private-f28-disasterrecovery
+- as: gcp-ipi-private-proxy-f28-disasterrecovery
```

Comment:
What is your test strategy? change the existing ones to cover the new configurations, or add new configuration coverage?

Reply:
change existing one to cover more configuration matric

Reply:
get it.